packages/library-base/src/commonMain/kotlin/io/realm/kotlin/ext/RealmDictionaryExt.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Realm Inc.
+ * Copyright 2023 Realm Inc.
```

Comment:
Nitpick: Should this file be named `RealmMapExt.kt` instead? Mostly because all the functions in here are not really extension methods on RealmDictionary but more factory methods.

Reply:
I think I will leave like this now and will re-evaluate naming later.